kedis/src/commonMain/kotlin/io/github/domgew/kedis/KedisClient.kt

```diff
@@ -237,6 +250,20 @@ public interface KedisClient : AutoCloseable {
         vararg key: String,
     ): Long
 
+    /**
+     * Sets a timeout on a key in seconds
+     *
+     * @param seconds the time in seconds
+     * @return Whether the operation was successful
+     *
+     * [https://redis.io/commands/expire/](https://redis.io/commands/expire/)
+     * */
+    public suspend fun expire(
+        key: String,
+        seconds: Int,
+        options: ExpireOptions
```

Comment:
a trailing comma enables one to reorder and add rows without having line changes on another unrelated line

```suggestion
        options: ExpireOptions,
```